### DISCUSSION TITLE
next_auth_providers_discord__WEBPACK_IMPORTED_MODULE_1__ is not a function

### DISCUSSION THREAD

**Discussion by Savers123** [nextauthjs/next-auth#5734] [Help] [answered]
I keep running into the following issue and I don't know where I'm going wrong

![image](https://user-images.githubusercontent.com/68947112/200116062-3c08bd6d-faf2-4978-a639-ea03171b97bb.png)

**Comment by fers4t** [comment]
Can you show us how you use ```DiscordOpts```? It seems you are trying to use it like a ```function```.

**Reply by Savers123** [reply]
Nvm, I got it working, after a few attemps

**Comment by Savers123** [comment, marked as answer]
I just restarted it from the beginning and it works now
